src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -53,6 +53,8 @@
 import org.kohsuke.stapler.StaplerRequest;
 
 public class MesosCloud extends Cloud {
+  private static final String DEFAULT_REJECT_OFFER_DURATION = "600000";
```

Comment:
is this in ms? to clarify you can do 

private static final String DEFAULT_REJECT_OFFER_DURATION_MS = "600000"; // 10 mins.